video/probe.go

```diff
@@ -2,20 +2,76 @@ package video
 
 import (
 	"context"
+	"fmt"
+	"strconv"
+	"strings"
 	"time"
 
 	"gopkg.in/vansante/go-ffprobe.v2"
 )
 
-type FFProbeClient interface {
```

Comment:
@mjh1 - i walked back on the changes to turn this into an interface. I'm envisioning `Probe` to be a generic utility function that can be called throughout our code base where required and didn't quite see a need to have it implemented as an `interface`.

Reply:
We need the interface and stub to allow this test to work though: https://github.com/livepeer/catalyst-api/pull/426/files#diff-917f0d52c54b320f45741c35aa8edf39fba81f03ce0f226dc4205c3c31a07e61R312 I didn't want to have a >2 minute sample video in the codebase so mocked up the ffprobe response instead.